client/src/app/routes/Landing.tsx

```diff
@@ -182,7 +182,7 @@ class Landing extends Feeds<RouteComponentProps<LandingRouteProps>, LandingState
                                         <h2>Latest messages</h2>
                                         <div className="feed--actions">
                                             <button
-                                                className="button--unstyled"
+                                                className="button--unstyled pause-button"
```

Comment:
Check if one of the util classes like margin-r-s or margin-r-t is a possible solution instead of adding pause-button class.